helm-chart/renku/values.yaml

```diff
@@ -53,6 +44,9 @@ global:
     gitlabClientId: renku-ui
     ## The client secret for authentication against gitlab
     gitlabClientSecret:
+    ## The secret for the keycloak client that will be used by amalthea
```

Comment:
I'd put this into `.Values.notebooks.jupyterServerClientId` and `.Values.notebooks.jupyterServerClientSecret` respectively. The init-realm script can pick it up from there as well so it doesn't really need to be a global value and the gateway is not involved here.